neo/Network/P2P/Payloads/ConsensusData.cs

```diff
@@ -45,5 +46,14 @@ public JObject ToJson()
             json["nonce"] = Nonce.ToString("x16");
             return json;
         }
+
+        public static ConsensusData FromJson(JObject json)
```

Comment:
For keep consistency, maybe keep the previous name `ParseFromJson` or just `FromJson` there as well?

Reply:
The static `FromJson` method in Block will hide member function `FromJson` in abstract base class BlockBase if use FromJson

Reply:
![image](https://user-images.githubusercontent.com/4393234/60876923-63be5b80-a26f-11e9-80ac-5046dcbfd3e8.png)

Reply:
@vncoelho Or we can use the following code in `Block` when using `FromJson` in `BlockBase`:
```C#
        public new static Block FromJson(JObject json)
        {
            Block block = new Block();
            ((BlockBase)block).FromJson(json);
            block.ConsensusData = ConsensusData.FromJson(json["consensus_data"]);
            block.Transactions = ((JArray)json["tx"]).Select(p => Transaction.FromJson(p)).ToArray();
            return block;
        }
```

Reply:
uhuaheuaheuahuea